components/lvgl_esp32_drivers/lvgl_spi_conf.h

```diff
@@ -70,22 +70,19 @@ extern "C" {
 #define SPI_BUS_MAX_TRANSFER_SZ (DISP_BUF_SIZE * 2)
 #endif
 
-#if defined (CONFIG_LVGL_TFT_USE_CUSTOM_SPI_FREQUENCY)
-#define SPI_TFT_CLOCK_SPEED_HZ (CONFIG_LVGL_TFT_SPI_CUSTOM_FREQUENCY * 1000 * 1000)
+#if defined (CONFIG_LVGL_TFT_USE_CUSTOM_SPI_CLK_DIVIDER)
+#define SPI_TFT_CLOCK_SPEED_HZ (80 * 1000 * 1000 / CONFIG_LVGL_TFT_CUSTOM_SPI_CLK_DIVIDER)
 #else
-
 #if defined (CONFIG_LVGL_TFT_DISPLAY_CONTROLLER_ST7789)
-#define SPI_TFT_CLOCK_SPEED_HZ  (24*1000*1000)
+#define SPI_TFT_CLOCK_SPEED_HZ  (20*1000*1000)
 #elif defined CONFIG_LVGL_TFT_DISPLAY_CONTROLLER_HX8357
 #define SPI_TFT_CLOCK_SPEED_HZ  (26*1000*1000)
 #elif defined CONFIG_LVGL_TFT_DISPLAY_CONTROLLER_SH1107
 #define SPI_TFT_CLOCK_SPEED_HZ  (8*1000*1000)
 #elif defined CONFIG_LVGL_TFT_DISPLAY_CONTROLLER_ILI9486
-#define SPI_TFT_CLOCK_SPEED_HZ  (24*1000*1000)
-#elif defined CONFIG_LVGL_TFT_DISPLAY_CONTROLLER_ILI9488
-#define SPI_TFT_CLOCK_SPEED_HZ  (40*1000*1000)
+#define SPI_TFT_CLOCK_SPEED_HZ  (20*1000*1000)
```

Comment:
Is 40MHz a reliable default clock for the ILI9488?

Reply:
Also, could you add an 'implicit' default value for the ILI9341 and ILI9488 displays?

Reply:
Sure, right now I’m not at the PC anymore, tomorrow I will take a look. Regarding ILI9488, 40MHz is perfectly working even on a breadboard.

Reply:
Thanks, I think 40MHz for all ILI9* controllers is ok on breadboards, I'm wiring my ILI9488 that way, maybe I'm just lucky :)

Reply:
Ok, reading some comments on the ILI9486, it seems that hardly reaches values above 30MHz, so until someone could test it, I would leave it to 20MHz (rounded from the 24MHz you set as default before).